src/components/Filter/Filter.test.jsx

```diff
@@ -0,0 +1,25 @@
+import { customRender } from "../../utils/testUtils";
+import { screen } from "@testing-library/react";
+import Filter from "./Filter";
+import userEvent from "@testing-library/user-event";
+import favArray from "../../data/favourites/favourites";
+
+const mockData = ["FOOD & DRINK", "CULTURE", "OTHER"];
+
+it("should render the filter component", () => {
+  const mockFunction = jest.fn();
+  const { container } = customRender(
+    <Filter favArray={mockData} setAllCardsArr={mockFunction} />
```

Comment:
use `favArray` as the prop's value, and we can delete `mockData` variable

Reply:
resolved